packages/web/src/alliance/Affirmations.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react'
+import { Image, ImageSourcePropType, StyleSheet, Text, View, ViewStyle } from 'react-native'
+import {
+  clabs,
+  clabsPreview,
+  laboratoria,
+  laboratoriaPreview,
+  wfp,
+  wfpPreview,
+} from 'src/alliance/images/index'
+import { H2, H3, H4 } from 'src/fonts/Fonts'
+import { NameSpaces, Trans, useTranslation } from 'src/i18n'
+import { Cell, GridRow, Spans } from 'src/layout/GridRow'
+import { useScreenSize } from 'src/layout/ScreenSize'
+import cLabsLogo from 'src/logos/cLabs-logo.png'
+import laboratoriaLogo from 'src/logos/laboratoria-dark.png'
+import wfpLogo from 'src/logos/wfp-logo.png'
+import Button, { BTN, SIZE } from 'src/shared/Button.3'
+import Photo from 'src/shared/Photo'
+import { fonts, standardStyles, textStyles } from 'src/styles'
+
+export default function Affirmations() {
+  const { t } = useTranslation(NameSpaces.alliance)
+  const { isMobile } = useScreenSize()
+
+  const orgStyle = isMobile ? mobileStyles : styles
+
+  return (
+    <View>
+      <GridRow
+        desktopStyle={standardStyles.sectionMarginTop}
+        tabletStyle={standardStyles.sectionMarginTopTablet}
+        mobileStyle={standardStyles.sectionMarginTopMobile}
+      >
+        <Cell span={Spans.full} style={isMobile && standardStyles.centered}>
+          <H3 style={isMobile && [textStyles.center, styles.mobileHeader]}>
+            {t('affirmations.smallTitle')}
+          </H3>
+          <H2 style={[standardStyles.halfElement, isMobile && textStyles.center]}>
+            {t('affirmations.title')}
+          </H2>
+        </Cell>
+      </GridRow>
+      <Exemplar
+        belief={<TransItalic i18nKey="affirmations.laboratoriaBelief" />}
+        copy={t('affirmations.laboratoriaCopy')}
+        image={laboratoria}
+        preview={laboratoriaPreview}
+        logo={<Image resizeMode="contain" source={laboratoriaLogo} style={orgStyle.laboratoria} />}
+        contentStyle={styles.laborStyle}
+      />
+      <Exemplar
+        belief={<TransItalic i18nKey="affirmations.wfpBelief" />}
+        copy={t('affirmations.wfpCopy')}
+        image={wfp}
+        preview={wfpPreview}
+        logo={<Image resizeMode="contain" source={wfpLogo} style={orgStyle.wfpLogo} />}
+        button={{
+          text: t('affirmations.wfpButton'),
+          href:
+            'https://medium.com/celoorg/how-to-design-for-all-stories-from-tanzania-refugees-8b34594d64ae',
```

Comment:
here you use `celoorg`, below you use `celohq`

Reply:
niice find. gracias.